### ISSUE TITLE
How to get raw text

### ISSUE THREAD

**Issue by Haiz14** [mosajjal/bard-cli#4]
Redirection to file gives output with styles
```bash
bard-cli "hello" >> bard.txt  
```
Gives
```text

^[[38;5;252m^[[0m^[[38;5;252m^[[0m  ^[[38;5;252mHello! How can I help you^[[0m^[[38;5;252m today?^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m^[[38;5;252m ^[[0m                              
```

How to get the raw files?

**Comment by mosajjal** [comment]
yep getting this fixed in the next release. the interactive mode will also have a component to write the previous answer into a file

**Comment by mosajjal** [comment]
fixed in a4f3b384ffccd34c75e5b1d9ad99c18384518a66
